jablib/src/test/java/org/jabref/logic/remote/RemoteUtilTest.java

```diff
@@ -24,10 +30,14 @@ void rejectPortsAbove16Bits() {
         assertFalse(RemoteUtil.isUserPort(65536), "Port number should be below 65535.");
     }
 
-    @Test
-    void acceptPortsAboveSystemPorts() {
-        // ports 1024 -> 65535
-        assertTrue(RemoteUtil.isUserPort(1024), "Port number in between 1024 and 65535 should be valid.");
-        assertTrue(RemoteUtil.isUserPort(65535), "Port number in between 1024 and 65535 should be valid.");
+    @ParameterizedTest
+    @CsvSource({
+            // ports 1024 -> 65535
+            "1024",
+            "65535"
+    })
+    void acceptPortsAboveSystemPorts(int port) {
+        String message = "Port number in between 1024 and 65535 should be valid.";
+        assertTrue(RemoteUtil.isUserPort(port), message);
```

Comment:
Remve `message` (see above)

Reply:
Keep it but move it directly into the assertTrue no need for a variable